lib/ae_mdw/db/format.ex

```diff
@@ -65,8 +72,11 @@ defmodule AeMdw.Db.Format do
 
     {status, auction} =
       case Name.locate_bid(state, plain_name) do
-        nil -> {:name, nil}
-        key -> {:auction, to_raw_map(state, key, Model.AuctionBid)}
+        nil ->
+          {:name, nil}
+
+        m_auction_bid ->
```

Comment:
Perhaps we can leave `locate_bid/2` to return `{:ok, auction} | nil` since it's better to pattern match on it and only update this line to handle it without any other changes?

Reply:
There are some other uses that would make it not straightforward and a non-tagged tuple is more symmetrical with `nil` that might also be returned.

I propose merging it as is and the legacy code can be refactored to work with ok/error tuples.